mne/time_frequency/tfr.py

```diff
@@ -1148,7 +1153,7 @@ def read_tfrs(fname, condition=None):
 
 
 def tfr_morlet(inst, freqs, n_cycles, use_fft=False,
-               return_itc=True, decim=1, n_jobs=1):
+               return_itc=True, decim=1, n_jobs=1, picks=None):
```

Comment:
while you're digging around in here, this function should also have a `verbose` argument